model_analyzer/config/generate/run_config_generator.py

```diff
@@ -46,11 +46,17 @@ def __init__(self, config, models, client):
         self._model_parameters = models[0].parameters()
         self._triton_server_env = self._models[0].triton_server_environment()
 
-        self._is_done = False
+        # This prevents an error when is_done() is checked befored the first next_config() call
+        self._pacg = PerfAnalyzerConfigGenerator(self._config,
+                                                 self._model_names,
+                                                 self._model_pa_flags,
+                                                 self._model_parameters)
+
+        self._model_configs_are_on_final_iteration = False
 
     def is_done(self):
-        """ Returns true if this generator is done generating configs """
-        return self._is_done
+        return self._pacg.is_done(
+        ) and self._model_configs_are_on_final_iteration
```

Comment:
did yapf do this?

Reply:
Yes. Wrapping it in `()`'s fixes the weird formatting